### PR TITLE
runelite-client: Consume hotkeys in HotkeyListener

### PR DESCRIPTION
**Pull request by Abextm**
KeyTyped consumer any non-modifier key while the hotkey is being pressed because there is no good way to determine if a KeyTyped event matches a Keybind due to it's keyCode being zeroed.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/config/Keybind.java [hunk 1]
```diff
@@ -39,15 +40,15 @@
 @EqualsAndHashCode
 public class Keybind
 {
-	private static final BiMap<Integer, Integer> modifierToKeyCode = new ImmutableBiMap.Builder<Integer, Integer>()
+	private static final BiMap<Integer, Integer> MODIFIER_TO_KEY_CODE = new ImmutableBiMap.Builder<Integer, Integer>()
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/config/Keybind.java]: .inverse is not very cheap afaik, you should probably declare static MODIFIER_TO_KEY_CODE_INVERSED or something

Comment (Abextm) [reply runelite-client/src/main/java/net/runelite/client/config/Keybind.java]: It is cached internally

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/config/Keybind.java]: I see, alright

File: runelite-client/src/main/java/net/runelite/client/util/HotkeyListener.java [hunk 2]
```diff
@@ -42,15 +42,28 @@
...
 
 	@Override
 	public void keyPressed(KeyEvent e)
 	{
 		if (keybind.get().matches(e))
 		{
+			boolean wasPressed = isPressed;
 			isPressed = true;
-			hotkeyPressed();
+			if (wasPressed)
```
Comment (whitehooder) [inline runelite-client/src/main/java/net/runelite/client/util/HotkeyListener.java]: It should be `if (!wasPressed)`, or else it still repeats when the key is held down and even skips the first press.

**Pull request merged**